src/Backend/Modules/Blog/Layout/Templates/Add.html.twig

```diff
@@ -68,7 +68,7 @@
                   <div class="panel-heading">
                     <p>
                       {{ 'lbl.Summary'|trans|ucfirst }}
-                      <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpSummary'|trans|ucfirst }}"></abbr><br>
+                      <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpSummary'|trans|ucfirst }}"></abbr><br>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320